EngineLayer/FdrAnalysis/PEPValueAnalysisGeneric.cs

```diff
@@ -13,13 +14,33 @@ namespace EngineLayer
 {
     public static class PEP_Analysis
     {
+        private static readonly string filepathSubstitue = "SubstitueFilePath";//this is only for unit test that have psms with no filename.
```

Comment:
If this is only for unit test. Add the filename in unit test is better.

Reply:
have to change too many unit tests.....

Reply:
i also think this should be removed, though it's fine to do this in a separate PR since this PR is a fairly high priority. when the code gets messy like this it's hard to follow and/or know what variables are dummy unit test variables and which are "real"